posts/2017-01-05-dat-next.md

```diff
@@ -1,9 +1,11 @@
 # Dat Next Preview Release
 
-Today we're announcing the release of Dat Next, our brand new Dat command line interface.
+Over the last few months we have been rebuilding the Dat command line tool and node modules to prepare for publishing datasets on our Dat registry. We've released a preview of this new command line tool you can install it now!
 
 Dat Next provides a more familiar command line interface, a lot of under the hood improvements, and an early preview of the Dat dataset registry commands. We are seeking feedback on Dat Next as we prepare to release it as our main command line tool for long term support.
 
+A big part of the Dat next development was a new Dat module for building Dat applications for the file system, [dat-node](https://github.com/datproject/dat-node). Before 2016 ended, we released the new API for dat-node. If you are interested in building Dat applications on the file system, check it out!
```

Comment:
Seems like the discussion for 'dat-node' could be introduced a little more straightforwardly, like 'The new dat